azure-pipelines.yml

```diff
@@ -235,36 +181,25 @@ jobs:
                           /p:SignType=$(_SignType)
         msbuildVersion: 15.0
 
-    - task: DotNetCoreCLI@2
-      displayName: 'Create NuGet packages'
-      inputs:
-        command: pack
-        projects: '$(solution)'
-        arguments: '--no-build --configuration $(buildConfiguration) --output $(Build.ArtifactStagingDirectory)'
-
     - task: MSBuild@1
       displayName: 'Sign nuget/snupkg packages'
       inputs:
         solution: eng/Sign.proj
         msbuildArguments: /t:SignBinaries 
-                          /p:SignAssetsDir=$(Build.ArtifactStagingDirectory)\
+                          /p:SignAssetsDir=$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries\
                           /p:SignNugetPackages=true
                           /p:SignType=$(_SignType)
         msbuildVersion: 15.0
     
     - task: CopyFiles@2
       displayName: Copy nupkg to publish
       inputs:
-        sourceFolder: $(Build.ArtifactStagingDirectory)
-        contents: 'Microsoft.Spark*.nupkg'
+        sourceFolder: $(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries
+        contents: |
+          **/*.nupkg
+          **/*.snupkg
         targetFolder: $(Build.ArtifactStagingDirectory)/Packages
-
-    - task: CopyFiles@2
-      displayName: Copy snupkg to publish
-      inputs:
-        sourceFolder: $(Build.ArtifactStagingDirectory)
-        contents: 'Microsoft.Spark*.snupkg'
-        targetFolder: $(Build.ArtifactStagingDirectory)/SymbolPackages
+        flattenFolders: true
```

Comment:
Just curious what does this do?

Reply:
It puts the files directly in the root folder (i.e. flattened) instead of having the same directory structure underneath.

So if you copy:

```
\a
   \b
        foo.exe
```

to a folder `\temp`, you would get `\temp\b\foo.exe`.  But here I want the `foo.exe` to be directly under `\temp`, and not to preserve the folder hierarchy.

Reply:
Cool. Good to know, thanks for explaining!